### ISSUE TITLE
Add osu players as pages?

### ISSUE THREAD

**Issue by jxu** [ppy/osu-wiki#4554]
This is outlined in my reddit post: https://www.reddit.com/r/osugame/comments/jwvttx/is_there_a_wiki_for_famous_players_like_smash/
Idk if it is appropriate for this wiki. The old wiki had a notable users page but that was very brief.

Also in my post I suggested moving the wiki from github to mediawiki because mediawiki is a lot easier to contribute to IMO (much more familiar for non-programmers who have never used git/github before). That may be its own issue.

**Comment by Nivalyx** [comment]
for the record someone have created [an article about "List of Notable People"](http://www.reddit.com/r/osugame/comments/6pj481/what_happened_to_the_list_of_notable_people/) already in the past but it was removed afterwards from the wiki due to content and quality concerns.

the thing about this is there are so many ways you can classify an user as "notable" -- would you call on a people to be "notable" based on their followers count? the amount of badges they have? the amount of ranked maps they have? their pp amount? or some other factor? and if any, in what threshold capacity would you do the cut between "notable" and "not notable"? all of these reeks of heavy subjectivity which doesn't really fit into a wiki that is intended to be a proper "knowledge base" in my opinion.

also the osu-wiki is hosted in github for a variety of reasons, including (but not limited to) its connectivity to other osu!-related repositories ([ppy/osu](http://github.com/ppy/osu), [ppy/osu-web](http://github.com/ppy/osu-web), etc.) and its content/quality moderation protocol. that being said however i suppose nothing is stopping you from creating an (unofficial) "famous osu! players" wiki in mediawiki or other wiki hosting sites; just make sure to be responsible and all in case you went through with it.

cheers :)

**Comment by jxu** [comment]
I understand your concerns about notability. We can copy Wikipedia's model by having a set of guidelines and also community votes if necessary on special cases. For example, a user is notable if they meet at least one of these criteria:
- Top 50 global / top x in country rankings
- On the winning team of major tournaments such as OWC
- Has special profile badge
etc. 
Similar guidelines can be applied to mappers, such as:
- Having at least 20 ranked maps
- Being the winner of a mapping tournament 
- Creating a map played over 1M times

Yes the criteria are subjective but that is tempered by having community consensus. Trust me, even Wikipedia gets it to work and people take notability a lot more seriously there. However, if such a player wiki does not fit the "official osu wiki" then the guidelines are moot. I hope that we can at least decide if players belong in the official wiki or not since that would make it a lot easier to get something going on my end.

**Comment by Ephemeralis** [comment]
Not a bad list to start with, though it poses some things to think about:

- Does someone have to be in the top 50 global for a specific period of time to be considered notable, or are they considered notable after hitting it once?
- What kind of threshold do we cut top players from different countries at? 5? 10?
- Mapper notability should be limited to plays/favorites, not total number of ranked maps submitted. Exceptions could be made for numbers in the triple digits, I guess, but 20 is probably too low to be considered anything especially notorious anymore.
- Significant participation in community tournaments as either a staffer or winner should be considered. What do we consider significant? Three wins? One win? How many tournaments does someone have to ref/contribute to before they're considered notable?

**Comment by jxu** [comment]
This was a quick list of thoughts so the exact numbers aren't worked out. However I think to hit top 50 once counts but I lean towards inclusivity over exclusivity in terms of who is considered notable. This doesn't include blatant cheaters. I think cheaters shouldn't be included as to not give them any attention, though maybe the notorious ones should get a page due to their high prominence already.
Top country rank threshold is a tricky one because some countries are vastly underrepresented in the leaderboards. Being from the US I can't say without bias if someone such as the top Indian player is considered notable or not. Perhaps the country threshold only applies for countries with at least a certain number of players, for example top 10 in the top 20 countries listed at https://osu.ppy.sh/rankings/osu/country
I'm not a mapper so we can adjust the mapper notability suitably. The last time I checked osu map statistics was in 2017 (https://osu.ppy.sh/community/forums/topics/631455) and I would put every mapper who has a top 20 most played map as notable. I can rerun the numbers if needed. 
I'm also not in the tourney community but I know a few people who are who could give better numbers on who counts as significant as a player and staffer. I do want to include everyone who has ever participated in an OWC and maybe we could also use profile badges as a measure of tournament success.

**Comment by jxu** [comment]
I still think a player wiki would be best with mediawiki templates to make infoboxes. I'm not familiar with how osu wiki does things admittedly but it looks like there's some basic YAML that could potentially be turned into useful infoboxes.

**Comment by venix12** [comment]
I believe people that are actively involved in running well known community projects (such as mentorship, spotlights, big tournaments like corsace for example etc etc) or generally community all-rounders should have their place in such pages as well. Not really sure how it'd be judged though.

**Comment by cl8n** [comment]
List of Notable People died because it got locked for vandalism and the few people allowed to edit from there didn't maintain it well. lots of complaints later it got removed and iirc the consensus was something like "this is too opinionated for wiki", which isn't surprising given the page tried to summarize people's entire impact on the game in a few sentences and listed people as if they were definitively more important than people not on the list. we're not making another "list" so I don't think there's muhc to worry about

imo pages about people are very fitting for a wiki and it's one of the major areas osu-wiki currently fails to cover. we reorganized the People category to prepare for branching out to community members, but nobody got around to writing full articles abt anyone, probably cuz high effort + stigma from list of notable people, but as long as you stay neutral and use a lot of references it'd be just like writing any other article

I don't think it's necessary to come up with hard definitions of who's allowed to be on the wiki either. if you can write a few good paragraphs about someone's involvement with osu! beyond private groups and such, there's a good chance other people would like to read about them

**Comment by Bazzadwarf** [comment]
I've wanted to make this suggestion for ages but never got around to it, I think it would be incredibly helpful to have this especially so for older users who have left a longstanding impact on the game but aren't wildly known to newer players. 

While there probably should be some user-specific guidelines for the wiki, I do believe that the wiki would be better off by having a proper user section.

**Comment by TicClick** [comment]
popularity and metrics don't always work well with notability or influence. as a general idea, it is good, but you'd miss things like James' Training, or James/Al-Azif's mapping that many mappers stem from, or [ASH/Larto's Jarto project](https://osu.ppy.sh/beatmapsets?q=jarto), or composers like tieff/La Cataline who have mapped their own music, or...

> I don't think it's necessary to come up with hard definitions of who's allowed to be on the wiki either. if you can write a few good paragraphs about someone's involvement with osu! beyond private groups and such, there's a good chance other people would like to read about them

this is basically the core, there are many small- and medium-sized things that were attributed to certain circles or individuals which are slowly vanishing. and yes, that was basically the old "List of notable people" article, a bunch of snippets about people who have left the mark in one way or another:

```
...
**SiLviA** was one of the best mouse players back in 2011, even achieving global rank 2 for a time. He is well known for his accuracy, alternating style and almost robotic snapping movement synonymous with mouse play. Mostly a Hidden only player, he has managed to get some crazy scores such as [http://osu.ppy.sh/b/40285 Hatsune Miku - Joker] or [http://osu.ppy.sh/b/98415 Hatsune Miku - With a Dance Number]. Even in 2014, he is still considered as one of the best mouse players osu! has ever had. Right now, he doesn't play osu! as heavily as he used to. He drops by from time to time and still manages to get high scores.
...
**James** was infamous for [http://osu.ppy.sh/forum/t/4280/ James' Training] with his signature "JAMES DIFFICULTY", in which many players referred to as a way of improving their skills in the early days of osu!. His beatmaps are also known to be unique at its own rights, deploying various types of mapping techniques based on the song used and difficulty settings. He had recorded to have done 125 mods in a single week, having run out of beatmaps to mod by the 5th day. He was also 1 of 3 people to be inducted into the osu! hall of fame.
...
**akrolsmir** created and maintained the [http://osu.ppy.sh/forum/t/55305 AIBAT] tool. This tool is one of the most more popular 3rd party tools along side osu!record and osu!stats.
**dukambe** created the [http://osu.ppy.sh/forum/t/209560 osu!Trainer] tool that suggests beatmaps based on current player's skill.
...
```
and yes, the article was a barely moderated, infrequently updated mess with names grouped by area of activity, but it was well worth it, as it was a piece of osu! history

**Comment by paper-toaster** [comment]
> ... but as long as you stay neutral and use a lot of references it'd be just like writing any other article

I don't think having user pages on the wiki would be a bad idea, but I'm wondering how exactly we would get references for this kinda stuff? afaik the wiki doesn't actually use many references (the ASC doesn't even detail how to credit sources) in general, and the deciding factor on what would be considered a "source" is quite vague. Not a dealbreaker, but still something to consider.

Also, if we were to go through with this, should articles that mention a notable player link to their actual user page or their wiki article? If so, the "History of osu!" pages are probably gonna need to be heavily updated.

> e: i swear we had another issue about this already but I can't find it. maybe it's a long-lost chat in discord i'm thinking of

I'm pretty sure a suggestion came up in #osu-wiki in like mid-july or something, but everyone agreed that it was basically a repeat of the list

**Comment by jxu** [comment]
References can come in several forms:
- archive.org snapshots. These are the most objective (though archive.org did mess up the date of some archives going back to ppv1)
- In game screenshots and osugame posts. These have the possibility of being wrong but are generally reliable. 
- Scores that are still in the osu database can be directly linked, ex. https://osu.ppy.sh/scores/osu/2177560145
- Firsthand or secondhand accounts by the players themselves or people who were around at the time. This is the least reliable but often a good source of info otherwise lost.

In general I have an archiving mindset in that it's ok if not everything is 100% correct as long as important info gets preserved. 
 

> I've wanted to make this suggestion for ages but never got around to it, I think it would be incredibly helpful to have this especially so for older users who have left a longstanding impact on the game but aren't wildly known to newer players.

I was in a discord chat with a new player who didn't even know who happystick was. This is the kind of thing that people memed about in the past :(

**Comment by paper-toaster** [comment]
has there been any further discussion thats happened abt this issue past this comment thread?

**Comment by jxu** [comment]
idk. well I might just start my own wiki since I have time right now

**Comment by paper-toaster** [comment]
> idk. well I might just start my own wiki since I have time right now

@jxu clayton did say that hes been [waiting for someone to pr something here](https://discord.com/channels/188630481301012481/218677502141399041/793216586323591199)

**Comment by jxu** [comment]
anyway I still think github PR process is too slow for and not suited for a community wiki. But it seems to be the process here

**Comment by TicClick** [comment]
it is totally appropriate to ask anyone on `#osu-wiki` in Discord for a review explicitly. for now, we go over pending PRs from time to time, and some things may be overlooked due to different definition of "time" for each one of us, especially if there is no clear indication of a pull being ready or waiting on someone's input

**Comment by TicClick** [comment]
to add more specifics as to why there is no progress, so far it's just a lot of ideas from different people. neither the final format, nor the assignee, are chosen, and no one wants to make a call on that, or take an initiative

**Comment by cl8n** [comment]
as much as I agree about the inexpressiveness of markdown and the annoyance of a review-first process for a community wiki, I think you're concluding too fast that this would limit your main idea. articles for players would be great and with the interest you've built up in this thread I don't think you have to worry about things moving too slow

> to add more specifics as to why there is no progress, so far it's just a lot of ideas from different people. neither the final format, nor the assignee, are chosen, and no one wants to make a call on that, or take an initiative

why does a "final format" need to be decided, just write a darn article at /wiki/People and we can go from there

**Comment by jxu** [comment]
@cl8n if it's mostly a small group of dedicated people writing pages, which is plausible, then the PR system is not so bad because they would know how to navigate it. The github system is bad for people not familiar with github.

In Markdown I would really like a way to do templates so we can have  consistent infobox display. At minimum we display a player's country, previous names, input method, and years active. We could also include peak rank and subjectively strengths (ex. DT player, HD player, all-rounder).

**Comment by paper-toaster** [comment]
> In Markdown I would really like a way to do templates so we can have consistent infobox display. At minimum we display a player's country, previous names, input method, and years active. We could also include peak rank and subjectively strengths (ex. DT player, HD player, all-rounder).

im not so savvy on the more technical details (although i very much would like to be), but at the current moment, i think tables could suffice *somewhat*, although it would look very ugle. I am curious as to whether or not infoboxes (or aligning images) in Markdown is possible, or is just a pain in the ass 🤔.

> ...  just write a darn article at /wiki/People and we can go from there

I think I could prob write up a sample article on Shigetora, but it would be limited to a couple youtube videos and my knowledge from like 2017 of research. I'll prob start writing sometime this week

**Comment by jxu** [comment]
I for some reason really want infoboxes for players, but I don't think it's possible to have one on the side like for wikis without resorting to HTML and maybe custom CSS.

**Comment by jxu** [comment]
I also thought about expanding the scope to include famous maps like Freedoms Dive and Big Black. I think they are noteworthy to have some background about them.

**Comment by cl8n** [comment]
html is currently stripped from wiki markdown, I've requested it be allowed for infoboxes in https://github.com/ppy/osu-web/issues/7031, not sure how that will go

i set a reminder for myself next weekend to get out some shorter articles abt players i remember, hopefully that helps get the ball rolling here. if we eventually want the rly big names' articles to be comprehensive, it's going to need a lot of research effort, so we should get together a group that can work on this on discord or somethn

**Comment by jxu** [comment]
I'm also concerned about how to do references in markdown. From what I've
seen the osu wiki isn't very strong on references, instead being a primary
source. It may need to be accomplished with footnotes.

**Comment by jxu** [comment]
Ok so now we can just copy this https://www.youtube.com/watch?v=cgGa-lNtsWE for cookiezi's article on notable plays right

**Comment by cl8n** [comment]
FYI infoboxes have been added now (but might not have their rules laid out per #5440, I haven't been following), if any other markdown features are needed for this let me know. I think GPR was interested in adding footnote support.

> Ok so now we can just copy this https://www.youtube.com/watch?v=cgGa-lNtsWE for cookiezi's article on notable plays right

whatever you feel works, just make sure to identify authors for anything not pulled from objective source (like curating notable plays)

**Comment by jxu** [comment]
I suddenly remembered this issue exists, and since we have https://osu.ppy.sh/wiki/en/History_of_osu%21/Online_rankings/osu%21, I wrote a very rough draft of a sample article, modeled off of smash bros wiki pages. 

Is there any consensus yet on if the player pages will be created? The reception seems leaning towards inclusion.

---

**Cookiezi** (currently **chocomint**, also **nathan on osu**) is an osu! standard player from South Korea that is widely regarded as one of or the greatest osu players of all time. Starting in 2009, he quickly improved and rose up the rankings, eventually attaining #1 global ranking in all three of osu's global leaderboards: Ranked Score, ppv1, and ppv2. He is well-known for his versatile reading, streaming ability, and accuracy, being able to DT FC many awkward older maps, play streams with great accuracy, and also conquer maps with technical and awkward patterns.

Infobox: country, join date, profile badges and playcount (as of a certain year?), twitch stream, former aliases (including old youtube accounts and alts) 

Sections

- History
  - Pre-ban
  - Ban, offline scores and livestreaming
  - Unban and return to competitive scene
- Notable scores by year (include r/osugame links and screenshots) 
- Tournament performances (table or list)

**Comment by Walavouchey** [comment]
I'm personally of the opinion that we should make articles while leaning on the inclusive side and iron out the formatting as we go along, because no consensus will be reached at this rate and we may have better chances (hopefully) at getting custom formatting in infoboxes once we get this project going.

If you can write a well-referenced article and pr it, we should just put it in. I don't myself have the time to write individual articles rn (although perhaps I might if literally no one steps up to write for a long while), but I don't see any reason any article would get outright rejected for now (especially for any obvious first candidates, of which there are plenty). I'm all for being more inclusive than exclusive, and hope that any eventual rejections would be for obvious reasons.

Notability can become more apparent once enough information of interest has been gathered for one person. If there is enough interesting information about a person to be able to write/draft a well-referenced and/or interesting article, while considering some of the criteria people have outlined here already and what will probably be iterated on as we go along, it's probably good to go.

So yeah a Cookiezi article would be a good start.

**Comment by jxu** [comment]
The obvious choice is anyone who has been #1 since pp system or was on the winning team of an OWC or anyone who was top 10 global. Those are the obvious choices. 

Is the Cookiezi article a good start? The problem with SSB wiki is that some pages aren't sourced that well.
